src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
You can just set the font styles to the whole page and then inherit where needed.
No need to use * selector for fonts. It is not optimized